benches/dnstap/mod.rs

```diff
@@ -1,15 +1,13 @@
 use bytes::Bytes;
 use criterion::{criterion_group, criterion_main, BatchSize, Criterion, Throughput};
 use vector::{
-    event::Event,
+    event::LogEvent,
     sources::dnstap::{schema::DnstapEventSchema, DnstapParser},
 };
 
 fn benchmark_query_parsing(c: &mut Criterion) {
-    let mut event = Event::new_empty_log();
-    let log_event = event.as_mut_log();
     let schema = DnstapEventSchema::new();
-    let mut parser = DnstapParser::new(&schema, log_event);
+    let mut parser = DnstapParser::new(&schema, &mut event);
```

Comment:
I think you actually dropped the event itself here. 😅

Reply:
Oops, wonder why `make check` and `make check-clippy` didn't scream about that.